### ISSUE TITLE
Documentation: Installation 

### ISSUE THREAD

**Issue by cwcromwell** [zdresearch/OWASP-Nettacker#169]
Two bigger changes that I think would make the installation chapter better:

1.) Fix use of the term "requirements." It appears that it is being used in different ways, which will confuse your end users:

* The summary at the beginning says there are multiple ways to install, including on this list "requirements," which is not exactly a mode of installation.
*The list of pre-requisite software appeared under the heading "requirements." I changed this to "prerequisites," but this calls attention to the issue more than fixing it. 
*There is a script called requirements that installs the actual components of OWASP nettacker. 

Here's what I'd like to do: 
* Move prerequisites higher up in the chapter, because they should be checked before installation begins. Maybe a new section up above installation methods would do it. 
* Remove "requirements" from that list of installation methods 

Let me know if I've misinterpreted anything here. 

2.) Add a support matrix at the top of the installation page, with more detailed information about supported versions of operating systems (e.g. what OSs are actually tested) and any info about problems with individual versions of Python. I would need some feedback for that. 

(I see that there's a line or two in the introduction, but more would be better.)

**Comment by Ali-Razmjoo** [comment]
Hi,

Thanks for your contribution and ideas, we are already testing the framework on [travis-ci](https://travis-ci.org/zdresearch/OWASP-Nettacker/) but it's not supporting much OSs for now, so that would be great to add the matrix, but should we add it now or wait for the official release?

Best Regards.

**Comment by shaddygarg** [comment]
Hey @cwcromwell, I was just going through the changes you made to the installation wiki and it looks clean and more human readable. I wanted to give a suggestion. [Docker Installation](https://github.com/zdresearch/OWASP-Nettacker/wiki/Installation#docker) is supposed to be another method of installation. The current structuring makes it look like a fallback option in case every other step doesn't work. I think we should reorder or reword it better. What do you think?

**Comment by cwcromwell** [comment]
I'll take a look.

**Comment by cwcromwell** [comment]
I think I've covered all the changes discussed above. 

To improve the flow, I did move some of the info, as proposed, into two new sections titled "before installation" and "after installation." 

I added a bit of information about the ability to cover operating systems and versions of Python, and we can update this with an actual matrix later, when it is easier to do so. 

I also took care of the issue with the Docker section, but let me know if that works as you had in mind, @shaddygarg.
